GetCertificate/certificate.headerInfo.ts

```diff
@@ -58,21 +57,19 @@ const emptyHeader: HeaderInfo = {
 
 export const getHeaderInfoForLanguage = (
   lang: O.Option<PreferredLanguageEnum>
-) => (certificate: O.Option<Certificates>): HeaderInfo => {
-  const language = pipe(
+) => (certificate: Certificates): HeaderInfo =>
+  pipe(
```

Comment:
if you use flow here you can omit the lang paramter. anyway, it's always better to not take monadic structures as params and use them in the caller.

Reply:
where the flow will infer the type?

Reply:
Regarding the monadic structures, I followed what we already did in [printer.ts](https://github.com/pagopa/io-functions-eucovidcerts/blob/ad5c8077928cf1014c2121bfd91ddf637d16dabe/GetCertificate/printer.ts#L108), but I got your point and I'll think about making a refactor of this.